pages/remote-jobs/filterComponent/Filter.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
The filterComponent should be located in the root component directory. 

When using Next.js  file-based routing, accessing the component via a URL like http://localhost:3000/remote-jobs/filterComponent/Filter is not intended behavior and should be avoided.